### PR TITLE
Feature/layout

### PR DESCRIPTION
**Pull request by ozanisgor**
- [x] Create footer with social icons and fix links 
- [x] Create navigation for other platforms and 'jobs' and fix links 
- [x] Change NEAR University logo path to relative link .svg and make it smaller
- [x] Create header and added NEAR logo

BEFORE:

10 02-22 before

AFTER:

image

### CODE REVIEW

File: web-app/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            d="M12 2C6.477 2 2 6.484 2 12.017c0 4.425 2.865 8.18 6.839 9.504.5.092.682-.217.682-.483 0-.237-.008-.868-.013-1.703-2.782.605-3.369-1.343-3.369-1.343-.454-1.158-1.11-1.466-1.11-1.466-.908-.62.069-.608.069-.608 1.003.07 1.531 1.032 1.531 1.032.892 1.53 2.341 1.088 2.91.832.092-.647.35-1.088.636-1.338-2.22-.253-4.555-1.113-4.555-4.951 0-1.093.39-1.988 1.029-2.688-.103-.253-.446-1.272.098-2.65 0 0 .84-.27 2.75 1.026A9.564 9.564 0 0112 6.844c.85.004 1.705.115 2.504.337 1.909-1.296 2.747-1.027 2.747-1.027.546 1.379.202 2.398.1 2.651.64.7 1.028 1.595 1.028 2.688 0 3.848-2.339 4.695-4.566 4.943.359.309.678.92.678 1.855 0 1.338-.012 2.419-.012 2.747 0 .268.18.58.688.482A10.019 10.019 0 0022 12.017C22 6.484 17.522 2 12 2z"
+            clipRule="evenodd"
+          />
+        </svg>
+      ),
+    },
+  ],
+};
+
+export default function Example() {
```
Comment (ryancwalsh) [inline web-app/components/Footer.tsx:47]: Please rename the function to `Footer`

File: web-app/components/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+export default function Example() {
```
Comment (ryancwalsh) [inline web-app/components/Header.tsx:1]: Please rename the function to `Header`

File: web-app/components/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
+/* eslint-disable react/jsx-props-no-spreading */
+const navigation = {
+  main: [
+    { name: 'Discord', href: 'https://discord.gg/k4pxafjMWA' },
```
Comment (ryancwalsh) [inline web-app/components/Footer.tsx:4]: Please move Discord and YouTube to the "social" section and use icons for them that match the styles of the others. I've never seen this inline SVG approach. We might want to do something simpler, like use https://fontawesome.com/search?q=discord&s=solid%2Cbrands (if you can find a free version that we're allowed to use) or a similar library / tool.

Comment (ryancwalsh) [reply web-app/components/Footer.tsx:4]: E.g. maybe https://ionic.io/ionicons or https://mdbootstrap.com/docs/standard/extended/social-media/ I'm forgetting other possibilities.

File: web-app/components/Header.tsx (hunk 2 above)
Comment (ozanisgor) [reply web-app/components/Header.tsx]: Done.

File: web-app/components/Footer.tsx (hunk 3 above)
Comment (ozanisgor) [reply web-app/components/Footer.tsx]: I used Font Awesome and move all socials to one place and move 'Jobs' to navbar

**Comment by ryancwalsh** [comment]
@ozanisgor This branch PR looks pretty good except that the header now is confusing since the Jobs link feels so out of place.

image

I wonder if we should just delete the Jobs link. 

And maybe play around with the footer and see whether you think this would look better:

- add a full-width horizontal line above it, sort of like how the header has a horizontal line along its bottom. Or make the entire footer background color non-white, such as a light gray. I don't know.
- Consolidate into one row, where there is the NEAR University logo and then all of the social icons all on the same row.

Another concern is that we don't want to have any dependencies in any project that lack licenses. I'm not sure about 
https://fortawesome.com/start It looks like we'd need to pay for it. Aren't there similar options that are free and offer licenses stating that _anyone_ can use them (even for-profit organizations)?

**Comment by ozanisgor** [comment]
> @ozanisgor Maybe the free version of https://fontawesome.com/plans has the social icons you want? You can check. fortawesome is a different spelling from fontawesome and is a different project. I'd never heard of fortawesome until I saw you use it. I think FontAwesome is popular.

@ryancwalsh Actually, I used the Font Awesome free version. I followed the steps in this website https://fontawesome.com/docs/web/use-with/react/ 

```bash
yarn add @fortawesome/fontawesome-svg-core
```

```bash
yarn add @fortawesome/free-solid-svg-icons
yarn add @fortawesome/free-regular-svg-icons
```

```bash
yarn add @fortawesome/react-fontawesome@latest
```

I think they are using `fortawesome` naming for installation. I thought they are using this naming because it is funny.

Maybe I am doing and seeing it wrong. I thought Font Awesome's free version works for us but I am not sure of course. If you suggest not to use Font Awesome, I may continue with something else as you suggest before.

**Comment by ryancwalsh** [comment]
@ozanisgor Interesting! I definitely assumed the different spelling was from a different library. But you're right. Ok, well, if it's free and if the license says that we're allowed to use it, and if it has the social icons we want, _great_!

**Comment by ryancwalsh** [comment]
@ozanisgor I just pulled this branch to have a fresh copy locally but still don't see any changes that I mentioned at https://github.com/NEAR-Edu/near-certification-tools/pull/7#issuecomment-1039434540 about the Jobs link or the footer. Have you played around with those yet to see what would look best? Screenshots (and your opinions) would be helpful. Thanks.

**Comment by ryancwalsh** [comment]
@ozanisgor Also, can you please delete the `h1` element from the page at http://localhost:3000/certificate/eb9c618b235649728883ce13d9ce1247 ?

image

We will come up with some better text later, in a future version of this page, once we're efficiently reading on-chain data.

**Comment by ozanisgor** [comment]
> @ozanisgor I just pulled this branch to have a fresh copy locally but still don't see any changes that I mentioned at [#7 (comment)](https://github.com/NEAR-Edu/near-certification-tools/pull/7#issuecomment-1039434540) about the Jobs link or the footer. Have you played around with those yet to see what would look best? Screenshots (and your opinions) would be helpful. Thanks.

@ryancwalsh Actually, yesterday changes were almost ready but I wanted your approval for the `Font Awesome` and push them all together because every time when I push and rebase I got some `git` errors and I need to re-apply my changes. Maybe I should record a loom video to show what I mean by this. For my next assignment I will try to record my every `git` step.

> Btw I can still remove the `jobs` if it is not looking good.

- [x] Consolidate into one row 
- [x] Add a full-width horizontal line above
- [x] Make the entire footer background color light gray

---

- ### _**This is how it look now**_

after-16 02 22_v1

---

- ### **_Alternative version (footer items order changed)_** 

after-16 02 22_v2

---

- ### **_I made a small change it is not in the seecreen shots which is aligning the icons center_**

image

File: web-app/components/Layout.tsx [hunk 4]
```diff
@@ -18,15 +20,9 @@ export default function Layout({ children }: { children: JSX.Element | JSX.Eleme
           referrerPolicy="no-referrer"
         />
       </Head>
-      <div>
-        <a href="/">NEAR University Certificate Browser</a>
-      </div>
+      <Header />
```
Comment (ryancwalsh) [inline web-app/components/Layout.tsx:23]: @ozanisgor This file looks so much cleaner now that Header and Footer are separate files. :-)

**Comment by ryancwalsh** [comment]
You're making progress.

1. I think it would look better if the entire footer were left-aligned. And the contents would be: NEAR Univ logo, then social icons, then Jobs link.

2. Are you sure you've pulled `develop` locally to make sure `develop` is up to date on your local machine and then check out your branch again and do `git rebase develop`? It sort of looks like this PR is showing lots of changes to `web-app/pages/index.tsx` that it shouldn't be showing since they already exist in the `develop` branch.  https://github.com/NEAR-Edu/near-certification-tools/blob/develop/web-app/pages/index.tsx

3. Can you fix the weird lack of padding that is happening on narrow screens such as for here?

image

**Comment by ryancwalsh** [comment]
@ozanisgor At https://near-certification-tools-tpq1.onrender.com/account/hatchet.testnet it would also be good to decrease the size of the heading and remove the spacing from above and below it:
image

And at https://near-certification-tools-tpq1.onrender.com/certificate/eb9c618b235649728883ce13d9ce1247, let's decrease the size of the certificate since it's too big and requires the visitor to scroll to much:

image

Maybe `max-height: 500px;`? And make sure it's responsive.

**Pull request merged**